addons/account/models/account_tax.py

```diff
@@ -593,6 +593,11 @@ def _check_company_consistency(self):
     def _sanitize_vals(self, vals):
         """Normalize the create/write values."""
         sanitized = vals.copy()
+
+        # Wrap plain text in <div> if description has no HTML tags
+        if 'description' in sanitized and not isinstance(sanitized['description'], bool) and not re.search(r'<[^>]+>', sanitized['description']):
+            sanitized['description'] = f"<div>{sanitized['description']}</div>"
```

Comment:
Please explain what is happening in the commit message
* `lxml.html` is parsing the string during `html_normalize`, which makes it a valid HTML by wrapping inside of `<p>` tags if it wasn't valid HTML
* on the other hand, if it was valid HTML, but wrapped with `<div>`, we remove it at the end of `html_normalize`

I don't know if this is good practice, but it is the only solution I can see in stable.
It has been added many years ago, and maybe not for good reasons
https://github.com/odoo/odoo/commit/c63ffed24d07d3d0294f443f71aa641cfd89bbba